README.md

```diff
@@ -1,83 +1,87 @@
+
+For more details on contributing refer to the [Contributing Guidelines](https://github.com/thsmi/sieve/blob/master/CONTRIBUTING.md).
+
+A big thank you to everyone who has [contributed and supported](CONTRIBUTORS.md) the [Sieve add-on](https://addons.thunderbird.net/addon/sieve/) project.
+
+If you want to support the [Sieve add-on](https://addons.thunderbird.net/addon/sieve/) project consider donating
+* code, patches or localizations
+* via Paypal [![PayPayl donate button](https://www.paypalobjects.com/en_US/i/btn/btn_donate_LG.gif)](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=EAS576XCWHKTC "Donate to this project using Paypal")
+* via Bitcoin [1GEnrRVobFYuqYj2opdvRQNtG3Z8znvfKw](bitcoin:1GEnrRVobFYuqYj2opdvRQNtG3Z8znvfKw?label=Donation%20for%20Sieve%20Addon)
 
 
 ## Questions and Bugs
 
-The best for questions is the public mailing list at https://groups.google.com/forum/#!forum/sieve-app or via mail sieve-app@googlegroups.com . But you can also send a private mail to schmid-thomas at gmx.net.
+The best place to ask questions is on the public mailing list at https://groups.google.com/forum/#!forum/sieve-app or by emailing sieve-app@googlegroups.com. However, you can also send a private email message to `schmid-thomas at gmx.net`.
 
-Concerning bugs please use the [issue tracker](https://github.com/thsmi/sieve/issues)
-or send a private email to schmid-thomas at gmx.net . You find more details on reporting bugs
-in the [Contributing Guidelines](https://github.com/thsmi/sieve/blob/master/CONTRIBUTING.md)
+Concerning bugs please use the [issue tracker](https://github.com/thsmi/sieve/issues) or send a private email to `schmid-thomas at gmx.net`. You find more details on reporting bugs in the [Contributing Guidelines](https://github.com/thsmi/sieve/blob/master/CONTRIBUTING.md)
 
 
-Give me 1-2 weeks time for a reply. If you did not receive a reply at all, it
-might be a good idea to check your spam filter.
+Please allow 1-2 weeks for a reply. If you did not receive a reply at all, it might be a good idea to check your spam filter.
+
 
 ## License
 
-The extension is free and open source software, it is made available to you
-under the terms of the [GNU Affero General Public License (AGPLv3)](http://www.fsf.org/licensing/licenses/agpl-3.0.html).
+The [Sieve add-on](https://addons.thunderbird.net/addon/sieve/) is free and open source software, it is made available to you under the terms of the [GNU Affero General Public License (AGPLv3)](http://www.fsf.org/licensing/licenses/agpl-3.0.html).
 
 Refer to [Licensing information](https://github.com/thsmi/sieve/blob/master/LICENSING_INFO.md) for details.
 
+
 ## Releases
 
-You are looking for the most recent release?
+You are looking for the most recent [Sieve add-on](https://addons.thunderbird.net/addon/sieve/) release?

```

Comment:
The extensive linking to addons.thunderbird.net looks very confusing and somehow strange to me.

Github is the official homepage of the addon. You can file feature requests, bugs view the source and get the most recent version. Thunderbird's addons site is just a simple an alternate mirror for the binaries. So there is no need for a user to visit thunderbird's addon site.

Reply:
Will change as you requested.